app-office/scribus/scribus-1.5.7.recipe

```diff
@@ -43,7 +43,7 @@ REQUIRES="
 	lib:libpng16$secondaryArchSuffix
 	lib:libpodofo$secondaryArchSuffix
 	lib:libpoppler$secondaryArchSuffix
-	lib:libpython3.7m$secondaryArchSuffix
+	lib:libpython3.9$secondaryArchSuffix
```

Comment:
Any specific reason to use python39 here over python3? (not sure how stable python39 is atm)

Reply:
Seems OK during build and usage today.